### PR TITLE
TST: troubleshoot npdev build

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Comment by jbrockmendel** [comment]
gentle ping @jreback this fixes the CI

**Comment by jreback** [comment]
you have an included profile file :(

remove

ping when pushed can just merge

**Comment by jreback** [comment]
thanks!

File: pandas/tests/dtypes/test_dtypes.py [hunk 1]
```diff
@@ -40,7 +40,12 @@ def test_equality_invalid(self, dtype):
         assert not is_dtype_equal(dtype, np.int64)
 
     def test_numpy_informed(self, dtype):
-        with pytest.raises(TypeError, match="data type not understood"):
+        # npdev 2020-02-02 changed from "data type not understood" to
+        #  "Cannot interpret 'foo' as a data type"
+        msg = "|".join(
+            ["data type not understood", "Cannot interpret '.*' as a data type"]
+        )
```
Comment (jorisvandenbossche) [inline pandas/tests/dtypes/test_dtypes.py]: I think we simply shouldn't test the message here, we are literally testing numpy here

**Comment by jorisvandenbossche** [comment]
Remember that in general, CI fixes should be backported

**Comment by jorisvandenbossche** [comment]
@MeeseeksDev backport to 1.0.x

**Pull request merged**